x-pack/legacy/plugins/ml/server/models/job_service/new_job_caps/field_service.ts

```diff
@@ -131,24 +131,24 @@ async function combineFieldsAndAggs(
   aggs: Aggregation[],
   rollupFields: RollupFields
 ): Promise<NewJobCaps> {
-  const textAndKeywordFields = getTextAndKeywordFields(fields);
+  const keywordFields = getKeywordFields(fields);
   const numericalFields = getNumericalFields(fields);
+  const ipFields = getIpFields(fields);
 
   const mix = mixFactory(rollupFields);
 
   aggs.forEach(a => {
     if (a.type === METRIC_AGG_TYPE) {
       switch (a.dslName) {
+        case ES_AGGREGATION.COUNT:
```

Comment:
It might be worth adding a comment to clarify the options in this switch, e.g. not knowing the internals for me it's not obvious why `ES_AGGREGATION.COUNT` does a `break` but `ES_AGGREGATION.CARDINALITY` does not.

Reply:
comments added in c13604c4393a1b3a36037c006c9294835bde0f51